core/pom.xml

```diff
@@ -246,6 +246,16 @@ THE SOFTWARE.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>xpp3</groupId>
+      <artifactId>xpp3</artifactId>
+      <version>1.1.4c</version>
```

Comment:
should this have a comment this is unused but for backwards compatibility?  Otherwise I'm not sure what this is here for.

Reply:
+1, would be nice to have a statement about it.